src/seth/libexec/seth/seth-namehash

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+### seth-namehash -- returns the ENS namehash of the provided name
+### Usage: seth namehash <name>
```

Comment:
Maybe worth adding an example that the usage is `seth namehash vitalik eth` and not `seth namehash vitalik.eth`

Reply:
Ah, I actually tweaked your implementation so the usage is `seth namehash vitalik.eth`, since it's more intuitive that way 

https://github.com/mds1/dapptools/blob/879c44645fe8d2fbcaeb0a99672bcbfb9406e162/src/seth/libexec/seth/seth-namehash#L19-L21